packages/stream_video/lib/src/call/call.dart

```diff
@@ -2141,27 +2141,38 @@ class Call {
     return result;
   }
 
+  /// Pins/unpins the given session to the top of the participants list.
+  /// The change is done locally and won't affect other participants.
   Future<Result<None>> setParticipantPinned({
     required String sessionId,
     required String userId,
     required bool pinned,
   }) async {
-    final result = await _session?.setParticipantPinned(
-          sessionId: sessionId,
-          userId: userId,
-          pinned: pinned,
-        ) ??
-        Result.error('Session is null');
+    _stateManager.setParticipantPinned(
+      sessionId: sessionId,
+      userId: userId,
+      pinned: pinned,
+    );
 
-    if (result.isSuccess) {
-      _stateManager.setParticipantPinned(
-        sessionId: sessionId,
-        userId: userId,
-        pinned: pinned,
-      );
-    }
+    return const Result.success(none);
+  }
 
-    return result;
+  /// Pins/unpins the given session to the top of the participants list for everyone in the call.
+  /// You can execute this method only if you have the `pin-for-everyone` capability.
```

Comment:
I think this is a user specific capability right?
```suggestion
  /// Users can execute this method only if they have the `pin-for-everyone` capability.
```